source/reference/about-mongosync.txt

```diff
@@ -0,0 +1,131 @@
+.. _about-mongosync:
+
+========================
+About ``mongosync``
+========================
+
+The ``mongosync`` binary is the primary process used in {+c2c-product-name+}. 
+``mongosync`` migrates data from one cluster to another and keeps the 
+clusters in continuous sync. You can use ``mongosync`` to create dedicated 
+analytics, development, or testing clusters that mirror your production 
+environment. 
+
+``mongosync`` enters different states depending on the requests it receives. 
+The current ``mongosync`` state determines which API operations you can run. 
+For more information on ``mongosync`` states, see 
+:ref:`mongosync States <c2c-states>`.
+
+.. note:: 
+
+   Most state transitions happen as the result of an API call. The exceptions 
+   are:
+
+   - The transition from ``COMMITTING`` to ``COMMITTED``
+   - The transition from ``REVERSING`` to ``RUNNING``
+
+How ``mongosync`` Works
+------------------------
+
+Initial Connection
+~~~~~~~~~~~~~~~~~~~
+
+``mongosync`` must create an initial connection to the source and destination 
+clusters before it can start to sync data. For more information, see 
+:ref:`c2c-connecting`.
+
+When ``mongosync`` connects to the source and destination clusters, it is in the 
+:ref:`IDLE <c2c-state-idle>` state.  
+
+During Sync 
+~~~~~~~~~~~~
+
+The :ref:`start <c2c-api-start>` endpoint begins the 
+:ref:`RUNNING <c2c-state-running>` state. 
+
+If you start a new sync, ``mongosync`` syncs data to the destination cluster. 
+If you make subsequent writes to the source cluster, ``mongosync`` applies them 
+to the destination cluster.  
+
+For every collection on the source, ``mongosync`` creates a corresponding 
+collection on the destination. 
+
+``mongosync`` temporarily alters some collection characteristics during 
+the ``RUNNING`` state and changes these back to their source 
+cluster characteristics during the ``COMMIT`` state. For more information, see
+:ref:`Commit Alters <c2c-commit-alters>`.
+
+Collection Copy
+'''''''''''''''
+
+Collection copy is the initial full data transfer that copies each collection 
+from the source cluster to the destination cluster. ``mongosync`` performs 
+collection copy by splitting each source collection into partitions, gathering 
+all partitions from all collections into one list, and copying all partitions 
```

Comment:
[question] can we either (A) talk about how the list of partitions is related to the process or (B) remove mentions of the list? I find it a bit confusing that the list is brought up but then not mentioned again

Reply:
Good point, I will go with (B) because I don't think the list is important for the users to know about. Changing to "``mongosync`` performs 
collection copy by splitting each source collection into partitions and copying all the partitions to their corresponding destination collections."